### ISSUE TITLE
ion-searchbar bug on 3.1.0, 3.1.1, 3.2.0 for non-latin languages.

### ISSUE THREAD

**Issue by kelvinhokk** [ionic-team/ionic#11566]
**Ionic version:**  (check one with "x")
[ ] **1.x**
[ ] **2.x**
[ X] **3.x**

**I'm submitting a ...**  (check one with "x")
[X ] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**

Replication environment:  Mac OS X (for PWA)
Different behaviours in Chrome  58.0.3029.96  and Safari Version 10.1 (12603.1.30.0.34) (see videos)

Currently I have replicated the bug on the ionic-preview-app on ionic versions 3.1.1, 3.1.0, 3.2.0

The bug does not happen on 3.0.1.

Description:  The key up on the ion-searchbar seems to be broken since 3.1.0 onwards for non latin keyboards.  For example I am using a chinese Hanyu pin yin keyboard to type "青山"  (qingshan).

However on Safari, the ion-searchbar goes like "qqqiqiqingqinqingqing sqing sqing shqing... ".  See video.
On Chrome, the ion-searchbar seems to dismiss the language bar intended to popup.

**Expected behavior:**

The ion-searchbar should work normally with the different language inputs.

See video.

**Steps to reproduce:**

1.  check out and build the ionic-preview-app and build 
2. ionic serve
3. scroll to the ion-searchbar page.
4. change input to chinese and type "qingshan".  notice on Chrome the input for chinese does not appear, and on Safari multiple letters appear.

5.  Change package.json ionic-angular dependency to 3.0.1
6.  remove node_modules and npm install again
7. notice that things work normally.

**Related code:**

```
insert any relevant code here
```
https://github.com/driftyco/ionic-preview-app

**Other information:**

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
insert the output from ionic info here
```
Cordova CLI: 6.5.0 
Ionic Framework Version: 3.0.1  (or 3.1.1 for the bug)
Ionic CLI Version: 2.2.2
Ionic App Lib Version: 2.2.1
Ionic App Scripts Version: 1.3.5
ios-deploy version: 1.9.1 
ios-sim version: 5.0.2 
OS: macOS Sierra
Node Version: v7.9.0

![bug on safari](https://cloud.githubusercontent.com/assets/7766953/25833239/50df65e6-34a2-11e7-95b4-78989247f1c7.png)
Bug on ion-searchbar on 3.1.0 and 3.1.1

![expected behaviour](https://cloud.githubusercontent.com/assets/7766953/25833238/5099329c-34a2-11e7-933a-bc381658439e.png)
Expected behaviour

**Comment by jgw96** [comment]
Hello, thanks for using Ionic! We will look into this.

**Comment by woodstream** [comment]
I found this bug a week ago too, it is strange that it just appear in PC，but work well in mobile

**Comment by miz-k** [comment]
I've also encountered the same symptom after using 3.1.0. when ion-searchbar had worked correctly with 2.x

**Comment by nishizhen** [comment]
we have the same bug~~~~

**Comment by puku0x** [comment]
I faced the same bug too during inputting Japanese.

I could solve this by replacing from `(input)` to `(change)` in https://github.com/driftyco/ionic/blob/master/src/components/searchbar/searchbar.ts#L37 .

**Comment by woodstream** [comment]
I had compared this version3.0.1 and version3.2.1 that I found to modify this function "Searchbar.prototype._inputUpdated" can fixed it

**Comment by jgw96** [comment]
Hello all! This is going to be fixed in the next release of ionic in this pr #11744

**Comment by danielsogl** [comment]
@jgw96 you linked to an issue ^^

**Comment by puku0x** [comment]
@jgw96 That's great ! Can't wait 😆
Thank you very much !!

**Comment by DavidRyu123** [comment]
@jgw96 It has a android and web issue too. When I type a Korean in ion-searchbar element, Korean alphabets show ㄱㅏㄴㅏㄱㄷㅏㄱ where it should be 가낙닥. 
Would you look into this issue too?
Thanks.

global packages:

    @ionic/cli-utils : 1.3.0
    Cordova CLI      : 7.0.1 
    Ionic CLI        : 3.3.0

local packages:

    @ionic/app-scripts              : 1.3.7
    @ionic/cli-plugin-cordova       : 1.3.0
    @ionic/cli-plugin-ionic-angular : 1.3.0
    Cordova Platforms               : android 6.2.3 ios 4.4.0
    Ionic Framework                 : ionic-angular 3.3.0

System:

    Node       : v7.7.2
    OS         : macOS Sierra
    Xcode      : Xcode 8.3.1 Build version 8E1000a 
    ios-deploy : 1.9.1 
    ios-sim    : 5.0.6

**Comment by road1987** [comment]
I faced the same problem too . when i am using chinese input method in input[text] . Looks like model would not change after blur event happen on the input box.  If  using english input method first ,  there is no problem .

**Comment by leon9608** [comment]
It is still a bug.

    @ionic/cli-utils : 1.4.0
    Cordova CLI      : 7.0.1 
    Ionic CLI        : 3.4.0

    @ionic/app-scripts              : 1.3.7
    @ionic/cli-plugin-cordova       : 1.4.0
    @ionic/cli-plugin-ionic-angular : 1.3.1
    Cordova Platforms               : none
    Ionic Framework                 : ionic-angular 3.3.0

    Node       : v6.11.0
    OS         : macOS Sierra
    Xcode      : not installed
    ios-deploy : not installed
    ios-sim    : not installed
    npm        : 3.10.10

**Comment by woodstream** [comment]
it had fixed on 3.4.0

**Comment by liujiawei520** [comment]
No,I alse have the issue in 3.4.0

**Comment by lxieyang** [comment]
@woodstream I'm still facing the same issue.

**Comment by KevinHongSpigit** [comment]
Still getting error with 3.4.0

**Comment by forfuns** [comment]
Still getting error with 3.6.1

**Comment by tscislo** [comment]
@DavidRyu123 did you managed to solve your problem? Was that fixed in recent version?

**Comment by DavidRyu123** [comment]
@tscislo yep. It's solved with update.

**Comment by tscislo** [comment]
Thanks it works for me as well in 3.4.2

**Comment by qwerqwermhc** [comment]
cli packages: (/usr/local/lib/node_modules)

    @ionic/cli-utils  : 1.19.0
    ionic (Ionic CLI) : 3.19.0

global packages:

    cordova (Cordova CLI) : 7.1.0 

local packages:

    @ionic/app-scripts : 1.3.7
    Cordova Platforms  : android 6.3.0 ios 4.5.4
    Ionic Framework    : ionic-angular 3.3.0

System:

    Node  : v9.0.0
    npm   : 5.5.1 
    OS    : macOS High Sierra
    Xcode : Xcode 9.1 Build version 9B55 

Environment Variables:
I would like to ask I have encountered this problem how can I update my order with what can I tell you

**Comment by skygragon** [comment]
I'm using 3.19.1, but still have this issue on IOS.

**Comment by FdezRomero** [comment]
@skygragon That's the version for the Ionic CLI, you have to update `ionic-angular` on your project:

```
npm install ionic-angular@latest --save
```
